apps/chat/src/store/share/share.epics.ts

```diff
@@ -387,6 +398,60 @@ const getSharedListingSuccessEpic: AppEpic = (action$, state$) =>
     filter(ShareActions.getSharedListingSuccess.match),
     switchMap(({ payload }) => {
       const actions = [];
+      const acceptedId = ShareSelectors.selectAcceptedId(state$.value);
+      const decodedAcceptedId = acceptedId && ApiUtils.decodeApiUrl(acceptedId);
+      const isNewResource = [
+        ...payload.resources.entities,
+        ...payload.resources.folders,
+      ].some((item) => item.id === decodedAcceptedId);
+
+      if (decodedAcceptedId && isNewResource) {
+        if (acceptedId.startsWith(ApiKeys.Conversations)) {
```

Comment:
Maybe it make sense to add `isConversationsId` and `isPromptId` (near to `isRootConversationsId` and `isRootPromptId` and reuse inside existed methods)